File: rest/call-feedback/instance-get-example-1/instance-get-example-1.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACba8bc05eacf94afdae398e642c9cc32d";
    string AuthToken = "{{ auth_token }}";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    var feedback = twilio.GetFeedback("CAbb9fe897bc46347cddd23ba42c94bdac");
    
    Console.WriteLine(feedback.DateCreated);
  }
}

File: wireless/devices/usage/instance-get-example-1/instance-get-example-1.js
// Download the **Next-Gen** twilio-node library from:
// twilio.com/docs/libraries/node#installation-nextgen
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'; // Your Account SID from www.twilio.com/console
var authToken = 'your_auth_token';   // Your Auth Token from www.twilio.com/console

var twilioLibrary = require('twilio');
var client = new twilioLibrary.Twilio(accountSid, authToken);

client.preview.wireless.devices('DEb8eff34b248d066a31c4a953134e183e')
  .usage().fetch(function(err, usage) {
    console.log(usage.period.start);
    console.log(usage.period.end);
    console.log(usage.dataCosts.total);
  });